### ISSUE TITLE
vipleague.la: popups

### ISSUE THREAD

**Issue by Alphamen16** [uBlockOrigin/uAssets#20472]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/19976), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.vipleague.la/la-liga/real-madrid-vs-rayo-vallecano-1-live-streaming`

### Category

popups

### Description

Pop-up blocker strick detects a pop-up trying to force itself through, as seen in this image: https://ibb.co/tCs0whT

Happens when you press the player, but it does not ALWAYS load, sometimes it takes a few refreshes to show up. The website it tries to call is actually blocked by UBO by default, but this is actually an href issue.

I was able to fix it with the following filter: `vipleague.*##[href*="dweatherbe"]`

However this does not stop other href's from loading. An Element inspection of the issue gave me a hint that it might be coming from an eventListener, but i was unable to make it go away completely without breaking something.

Ideally there would be a way to create a filter to disable all 3rd party href's? Something like `example.com$href, 3p` I KNOW that does not exist but there might a filter that does this and i'm just not skilled enough to get it. If i just block all href on the website, i will break core functionality.

Thanks in advance! Hope the issue can be reproduced :)

### Other extensions used

Pop-up blocker strict; TWP (translate web pages)

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.53.0
Firefox: 119
filterset (summary):
 network: 104865
 cosmetic: 41803
 scriptlet: 22483
 html: 1291
listset (total-discarded, last-updated):
 added:
  ublock-annoyances: 5708-13, now
 default:
  user-filters: 83-0, never
  ublock-filters: 36014-124, now
  ublock-badware: 7434-5, now
  ublock-privacy: 651-0, now
  ublock-unbreak: 2180-0, now
  easylist: 72481-702, now
  easyprivacy: 32836-726, now
  urlhaus-1: 10925-0, now
  plowe-0: 3721-1, now
  ublock-quick-fixes: 132-1, now
filterset (user): [array of 70 redacted]
trustedset:
 added: [array of 2 redacted]
 removed:
  edge-scheme
  wyciwyg-scheme
switchRuleset:
 added: [array of 135 redacted]
hostRuleset:
 added: [array of 452 redacted]
urlRuleset:
 added: [array of 104 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings:
 selfieAfter: 11
 trustedListPrefixes: ublock- user-
 userResourcesLocation: [redacted]
supportStats:
 allReadyAfter: 784 ms (selfie)
 maxAssetCacheWait: 75 ms
popupPanel:
 blocked: 11
 network:
  d34gjfm75zhp78.cloudfront.net: 1
  dweatherbe.org: 2
  nandweandthe.org: 4
  starszoom.re: 2
  tabbingtremors.life: 2
 extended:
  ##.bg-dark.ratio > .position-absolute
  ##.m-1.btn-danger.btn
  ##+js(abort-on-property-write, _pop)
  ##+js(abort-current-script, setTimeout, admc)
  ##+js(abort-current-script, JSON, break;case)
  ##+js(abort-current-script, JSON.parse, break;case $.)
  ##+js(abort-current-script, parseInt, break;case $.)
  ##+js(addEventListener-defuser, , break;case $.)
  ##+js(abort-current-script, Math, break;case $.)
  ##+js(abort-current-script, eval, replace)
  ##.Ad-Container:not(.adsbygoogle)
```
</details>

**Comment by stephenhawk8054** [comment]
> Other extensions used
>
> Pop-up blocker strict; TWP (translate web pages)

Can you turn off all other extensions and test again?

**Comment by Alphamen16** [comment]
@ghajini yeah, seems that it was an aeld filter as suspected, the site was added there! I'll be sure to flag here once they change stuff again, thank you for the help :)

**Comment by Alphamen16** [comment]
Hey @ghajini issue is still present, after a bit more digging i think i fixed it using a nosiif rule, as seen in this image: https://ibb.co/H2CmpP2

You can see that both niaomea.me and vipleague are calling it somewhere in the code, i noticed because if you click on the iframe of the player, every few seconds the pop-up appears. So by removing intervals associated with href (i already knew that's how the links keep poping up) it seems to have fixed it.

Not sure if this is perfect or even worth adding but it's just another 2 cents

**Comment by Alphamen16** [comment]
Will do!
